src/auth/auth.controller.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Yea so about all the any's on Auth....

This project is not returning users with any other relations, so we can just set the @getMe type back to <User> from @prisma/client without much trouble.

A little background, in the JWT strategy, when you validate the user and return it in the response, sometimes you want to INCLUDE relational data, such as an Admin profile, or some other related user data. When you do this, TypeScript will yell at you because the user object has been expanded beyond what prisma calls User.

In cases like this (we might never need it for this project) You would have to created your own user type that extends Prism's User as well as any other related types you are joining together and then use that extended type in the @getMe user decorator.

Just a heads up sort of thing. the use of any is gross, but sometime you gotta move fast! That's my fault Sorry!